python/doc/usecases/use_case_stiffened_panel.rst

```diff
@@ -0,0 +1,207 @@
+
+The output of interest is:
+
+- :math:`(N_{xy})_{cr}`: the critical shear force (N)
+
+We assume that the input variables are independent except the :math:`f_1` and
+:math:`f_2` for which we measure a Spearman correlation of :math:`\rho^S_{12}=-0.8`,
+modelled using a :class:`~openturns.NormalCopula`.
+
+The critical load :math:`(\tau_{xy})_{cr}` of a stiffened panel subject to shear load is given by
+
+.. math::
+  (\tau_{xy})_{cr}=k_{xy}\frac{\pi^2D}{b_0^2t_s}
+
+where
+
+- :math:`a` is the width of the panel;
+
+- :math:`b_0` is the width between too consecutive stiffener feet;
+
+- :math:`t_s` is the thickness of the panel main surface;
+
+- :math:`E_s` is the Young modulus of the panel main surface;
+
+- :math:`\nu_s` is the Poisson coefficient of the panel main surface;
+
+- :math:`D=\frac{E_st_s^3}{12(1-\nu_s^2)}` is the bending coefficient of the
+  panel main surface;
+
+- :math:`k_{xy}` is the load factor associated to shear buckling. It is given as
+  a function of :math:`\frac{b_0}{a}` through the empirical relation
+  :math:`k_{xy}=5.35 + 4\left(\frac{b_0}{a}\right)^2`
+
+It is more convenient to use the shear force :math:`N_{xy}` instead of the shear
+stress component :math:`\tau_{xy}`. It leads to the relation:
+
+.. math::
+   N_{xy}=q_1+q_c
+
+where :math:`q_1` abd :math:`q_c` are the shear fluxes in the panel main surface
+and its stiffener. They are given by:
+
+.. math::
+   q_1=\tau_{xy}t_s=2G_sh_0t_s\frac{\partial^2w}{\partial x\partial y}
+
+and
+
+.. math::
+  q_c=\frac{G_ct_cp}{\ell}\left[h-2h_0+\frac{h_c}{2p}(f_1-f_2)\right]\frac{\partial^2w}{\partial x\partial y}
+
+where
+
+- :math:`G_s=\frac{E_s}{2(1+\nu_s)}` is the shear modulus of the panel main
+  surface;
+
+- :math:`\frac{\partial^2w}{\partial x\partial y}` is the torsion strain of the
+  panel main surface;
+
+- :math:`G_c=\frac{E_c}{2(1+\nu_c)}` is the shear coefficient of the stiffener;
+
+- :math:`t_c` is the thickness of the stiffener;
+
+- :math:`h_c` is the distance between the mean surfaces of the stiffener hat and
+  foot;
+
+- :math:`h=h_c+\frac{t_c+t_s}{2}` is the distance between the mean surfaces of
+  the stiffener hat and the panel main surface;
+
+- :math:`f_1` is the width of the foot of the stiffener;
+
+- :math:`f_2` is the width of the hat of the stiffener;
+
+- :math:`p` is the half-widht of the stiffener;
+
+- :math:`R` is the radius of the circular part of the stiffener;
+
+- :math:`\theta` is the angle of the circular part of the stiffener;
+
+- :math:`\ell` is the length of the stiffener flank;
+
+- :math:`d=\frac{\ell-f_2}{2}-R\theta- :math:` is the half-lenght of the straight
+  part of the flank of the stiffener;
+
+- :math:`A=\ell t_c` is the area of the section of an half-ondulation;
+
+- :math:`\bar{A}=A+pt_s+\frac{1}{2}(f_1-f_2)t_c` is the area of the section of
+  the full panel (main surface and stiffener) bounded by :math:`p`;
+
+- :math:`h_0=\frac{1}{2\bar{A}}\left[A(h_c+t_c+t_s)+\frac{1}{2}t_c(f_1-f_2)(t_c+t_s)\right]`
```

Comment:
Square brackets are usually used only with a particular meaning. Here, there is no specific interest to use them.
```suggestion
- :math:`h_0=\frac{1}{2\bar{A}} \left(A(h_c+t_c+t_s)+\frac{1}{2}t_c(f_1-f_2)(t_c+t_s)\right)`
```

Reply:
And centering it will make it easier to read.